appeals/web/src/server/appeals/appeal-details/__tests__/appeal-details.test.js

```diff
@@ -1050,6 +1123,60 @@ describe('appeal-details', () => {
 					'/costs/lpa/select-document-type/2">Add</a>'
 				);
 			});
+
+			it('should render a costs decision row in the case documentation accordion with "Uploaded" status, empty received column, and "add" and "manage" action buttons, if there are documents in the costs decision documents folder', async () => {
+				const appealId = 3;
+				nock('http://test/')
+					.get(`/appeals/${appealId}`)
+					.reply(200, {
+						...appealData,
+						costs: {
+							decisionFolder: {
+								caseId: 1,
+								id: 3,
+								path: 'appeal_costs/decision',
+								documents: [appealCostsDocumentItem]
+							}
+						}
+					});
+
+				const response = await request.get(`${baseUrl}/${appealId}`);
+
+				expect(response.statusCode).toBe(200);
+
+				const element = parseHtml(response.text);
+
+				expect(element.innerHTML).toMatchSnapshot();
```

Comment:
Don't think you need a whole page match given you are looking for the elements you need :)

Reply:
Ideally we should only have 1 whole page match